### ISSUE TITLE
Default Language

### ISSUE THREAD

**Issue by toomanylogins** [melquibrito/source-code-editor-tinymce-plugin#3]
Hello thank you for this plugin. Makes editing the source to easier. By default plugin seems to be reverting to the Spanish language when it should be English ?
Regards
Paul

**Comment by mr-shady** [comment]
i have the same problem

**Comment by prathamVaidya** [comment]
I don't think this plugin is maintained any further. For anyone landing here in future, Try  [supercode](https://github.com/prathamVaidya/supercode-tinymce-plugin) plugin that I created for tinyMCE editor. It fixes all these issues and offers more.

![chrome-capture-2024-4-7 (1)](https://github.com/prathamVaidya/supercode-tinymce-plugin/assets/61202986/eb519154-ac86-423a-8d63-450f174c7e4b)
